devicetypes/smartthings/testing/smartsense-virtual-open-closed.src/smartsense-virtual-open-closed.groovy

```diff
@@ -0,0 +1,412 @@
+/**
```

Comment:
What is that file? It doesn't look as part of DTH for ORVIBO. Remove it from PR is not necessary

Reply:
That those were added by mistake,and I have deleted it.